Source/AliveLibAO/PauseMenu.cpp

```diff
@@ -384,6 +392,24 @@ void PauseMenu::VUpdate_44DFB0()
 
                     auto last_pressed = Input_GetLastPressedKey_44F2C0();
                     char lastPressedKeyNT[2] = { last_pressed, 0 };
+
+#if ORIGINAL_PS1_BEHAVIOR // OG Change - Exit save menu using controller
+                    if (last_pressed == VK_ESCAPE || last_pressed == VK_RETURN) // Keyboard ESC or ENTER
```

Comment:
this whole block is duplicated earlier - should prob be in its own func

Reply:
Should I make the function in AE or AO?

Reply:
Oh yeah I forgot everything exists twice - yeah just leave it copy pasted for now I guess